code-styling/mysql/README.md

```diff
@@ -0,0 +1,574 @@
+---
+permalink: /code-styling/mysql/
+---
+
+# Dealerdirect style guide: MySQL
+
+This guide defines the MySQL coding standards we use at Dealerdirect. The intent
+of this guide is to help developers create MySQL queries that are readable,
+portable, easy to maintain and bug-free.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119][rfc2119].
+
+## Table of Content
+
+- [Files](#files)
+
+- [Inline queries](#inline-queries)
+
+  - [Queries in PHP files](#queries-in-php-files)
+
+- [Database design](#database-design)
+
+  - [Tables](#tables)
+  - [Columns](#columns)
+  - [Foreign keys](#foreign-keys)
+
+- [Formatting](#formatting)
+
+  - [Select queries](#select-queries)
+  - [Joins](#joins)
+  - [Delete queries](#delete-queries)
+  - [Insert queries](#insert-queries)
+  - [Update queries](#update-queries)
+  - [Other select criteria](#other-select-criteria)
+  - [Grouping](#grouping)
+  - [Ordering](#ordering)
+  - [Limit](#limit)
+
```

Comment:
I think the subject 'subqueries' is missing.
( added as duplicate, because my first omment with this text doesn't show in the conversations)